src/components/UI/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link, useLocation } from "react-router-dom";
+import { CTA, SearchBox } from "../../../components";
+import "./Header.css";
+
+export const Header = () => {
+	const { pathname } = useLocation();
+	return (
+		<header className="header">
+			<Link className="brand__text grid-center" to="/">
+				Magnificent Notes
+			</Link>
+			{pathname === "/" ? null : <SearchBox />}
```

Comment:
> You can use && instead of this

Inspired from this article. You may check as well. 
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx